### DISCUSSION TITLE
how to filter posts with tags field by service value, how to api return data for posts with tags=service?

### DISCUSSION THREAD

**Discussion by Martian95** [directus/directus#19243] [Feature Requests]
### Summary

how to filter posts with tags field by service value, how to api return data for posts with tags=service?

### Basic Example

for example i have a list of posts stored in posts collection , each post has its own tags , is there a way that when i call the api it will return posts with a tag field with a specific value according to my needs ?

### Motivation

its easy to interact with api

### Detailed Design

no

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

no

### Alternatives

no

### Adoption Strategy

no

**Comment by DanielBiegler** [comment]
Hi there! Of course you can filter, see the [Filter-Documentation here](https://docs.directus.io/reference/filter-rules.html). See the "Filter Operators" for lots of different ways to filter your data. You might also be interested in the [Global Query Parameters](https://docs.directus.io/reference/query.html) which enable you to filter deeply nested fields.
